app/services/configuration/user_answers_default.py

```diff
@@ -12,10 +21,81 @@ def get_default_your_home_answers():
     return YourHomeAnswers(people_in_house=3, postcode="0000", disconnect_gas=False)
 
 
+def get_default_heating_answers():
+    """
+    Return a default 'heating' answers object.
+    """
+    return HeatingAnswers(
+        main_heating_source="Heat pump",
+        heating_during_day="Never",
+        insulation_quality="Moderately insulated",
+    )
+
+
+def get_default_hot_water_answers():
+    """
+    Return a default 'hot water' answers object.
+    """
+    return HotWaterAnswers(
+        hot_water_usage="Average",
+        hot_water_heating_source="Electric hot water cylinder",
+    )
+
+
 def get_default_cooktop_answers():
     """
     Return a default 'cooktop' answers object.
     """
     return CooktopAnswers(
         cooktop="Electric (coil or ceramic)",
     )
+
+
+def get_default_driving_answers():
+    """
+    Return a default 'driving' answers object.
+    """
+    return DrivingAnswers(
+        vehicle_size="Medium",
+        km_per_week="200",
+        vehicle_type="Electric",
```

Comment:
A typical user for this tool is assumed to be replacing something that is more or less end-of-life, so we are providing a forward-looking view.

The script that optimizes electricity plans for an electrified household currently makes use of the 'default' answers, so they align with an electrified house. This will be reworked in a future PR when we add something similar for natural gas.